### ISSUE TITLE
Button text is hard to read in focused state

### ISSUE THREAD

**Issue by robertknight** [hypothesis/client#2610]
The color of button labels in buttons that have keyboard focus has changed to black, which makes it hard to read.

<img width="488" alt="Button focused state" src="https://user-images.githubusercontent.com/2458/95171528-e9e88b00-07ad-11eb-819c-13da44dd9e4b.png">

Tested in Safari 14.0 and Chrome 87 with Hypothesis client version 1.549.0. The issue is not present in v1.548.0, so I suspect this may be caused by f727de42180e5fd6e37befccf685204579f1553a.
